joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/CallTests.scala

```diff
@@ -196,4 +196,34 @@ class CallTests extends CSharpCode2CpgFixture {
     }
   }
 
+  "null-coalescing operator" should {
+    val cpg = code("""
+        |namespace Baz
+        |{
+        |    public class Foo
+        |    {
+        |        private readonly IDatabase db;
+        |
+        |        public async AnyValue GetValue(string x)
+        |        {
+        |            var value = await db.get(x) ?? new AnyValue();
+        |            return value;
+        |        }
+        |    }
+        |}
+        |""".stripMargin).moreCode("""
+        |namespace Baz;
+        |
+        |public interface IDatabase {
+        | public AnyValue get(string x) {}
+        |}
+        |""".stripMargin)
+
+    "resolve methodFullName" in {
```

Comment:
I would like AST tests here too, please. Specifically to verify the coalescing operator.